src/main/java/myessentials/utils/ChatUtils.java

```diff
@@ -36,6 +51,46 @@ public static void sendChat(ICommandSender sender, String msg, Object... args) {
     }
 
     /**
+     * Parses a legacy text
+     * @param message A formatted legacy text
+     * @return The parsed text
+     */

```

Comment:
You don't have to add these comments. It adds no more information than the name of the method and its parameters.